### ISSUE TITLE
Stopping a VM doesn't update the UI for several seconds 

### ISSUE THREAD

**Issue by dustinsoftware** [Faithlife/sdc-portal#2]
It would be helpful to have some kind of message or indication that clicking "Stop" successfully sent the request to the server. Currently, I have to wait for the UI to poll for the server's status to see that something changed.

**Comment by Schoonology** [comment]
Agreed! I'm currently working on WebSocket support, which will help this immensely, as will updating the UI to indicate "pending" actions.
